Lib/test/test_posix.py

```diff
@@ -1622,6 +1622,23 @@ def test_setsigmask_wrong_type(self):
                             os.environ, setsigmask=[signal.NSIG,
                                                     signal.NSIG+1])
 
+    def test_start_new_session(self):
+        import subprocess
+        # For code coverage of calling setsid().  We don't care if we get an
+        # EPERM error from it depending on the test execution environment, that
+        # still indicates that it was called.
+        try:
+            output = subprocess.check_output(
+                    [sys.executable, "-c",
+                     "import os; print(os.getpgid(os.getpid()))"],
```

Comment:
Why does this code gets the process group id instead of the session id? While the process group is also changed by `setsid()`, I don't see why we would want such an indirect check if we have `os.setsid/os.getsid`.